stream/pbf.js

```diff
@@ -0,0 +1,68 @@
+
+/**
+  The pbf parser is responsible for configuring and executing a pbf parser and
+  returning a valid readable stream.
+**/
+
+var fs = require('fs'),
+    pbf2json = require('pbf2json'),
+    settings = require('pelias-config').generate(),
+    features = require('../config/features');
+
+// Create a new pbf parser stream
+function createPbfStream( opts ){
```

Comment:
please document what opts is expected to have, like

/**
 * @param {object} opts
 * @param {string} opts.file ...
 * @param {string} opts.leveldb ...
 * etc...
 */